generators/client/templates/vue/tsconfig.json.ejs

```diff
@@ -21,14 +21,11 @@
         "paths": {
             "@/*": ["src/main/webapp/app/*"]
         },
-        "importHelpers": true,
-        "allowJs": true
+        "importHelpers": true
```

Comment:
c'est bien voulu ce changement ?

Reply:
According to https://github.com/cypress-io/cypress/issues/7503 we have to use a workaround while Cypress 5.0 is available

Reply:
ok